src/core/sent_packet_metadata.c

```diff
@@ -32,19 +32,19 @@ QuicSentPacketMetadataReleaseFrames(
 #pragma warning(push)
 #pragma warning(disable:6001)
```

Comment:
Do we still need these warning disables here?

Reply:
Nevermind I thought these were different. We probably still do.

Reply:
Yes, I believe so, because SAL doesn't understand the concept of ref counted deletes.

Reply:
6001 was actually uninitialized false positives. So definitely still needed.